reference/constraints/Email.rst

```diff
@@ -7,7 +7,7 @@ cast to a string before being validated.
 +----------------+---------------------------------------------------------------------+
 | Applies to     | :ref:`property or method <validation-property-target>`              |
 +----------------+---------------------------------------------------------------------+
-| Options        | - `strict`_                                                         |
+| Options        | - `mode`_                                                           |
```

Comment:
Shouldn't the `strict` option have been annotated as "deprecated in Symfony 4.1" rather than completely replaced?